ReactiveTask/Task.swift

```diff
@@ -545,16 +545,18 @@ public func launchTask(taskDescription: Task, standardInput: SignalProducer<NSDa
 						dispatch_group_leave(group)
 					}
 
-					task.launch()
+					rawTask.launch()
 					close(stdoutPipe.writeFD)
 					close(stderrPipe.writeFD)
 
+					observer.sendNext(.Launch(task))
```

Comment:
We should ensure that `Launch` is sent as the first event. As it is here, I think it's a race. :confused: